tests/slenkins/slenkins_node.pm

```diff
@@ -96,7 +96,7 @@ sub run {
         systemctl stop $firewallservice
         systemctl restart sshd
     ";
-    script_output($conf_script, 100);
+    script_output($conf_script, 1500);
```

Comment:
It's a huge increase! I'm fine with this, but how can we be sure that *real* failing tests will not take too much time to fail?

Reply:
I agree, maybe we should double (or triple?) the timeout first. But this script_output() block is only for setting up SUT machine to have proper repos, configuration of network and so on. Anyway, 25min is not too long especially if you are adding 50+ repos at once.

The test itself is executing from different place anyway, and there is already longer timeout 2000s so I would say it will not hurt, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/8196c39da288cc6805e97623cc7934d38e68532f/tests/slenkins/slenkins_control.pm#L211